### PR TITLE
Improve initialization

### PR DESCRIPTION
**Pull request by conradoverta**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/App.java [hunk 1]
```diff
@@ -174,38 +175,37 @@ public static void main(String[] args) {
...
+
+      // Initialize data access
+      DAOSet daos = DAOSet.fromServices(services);
+
+      // Initialize telemetry
+      initializeTelemetryBasedOnConfig(config);
+
+      // Initialize cron jobs
+      CronJobUtils.initializeBasedOnConfig(
+          config, services.authService, services.roleService, services.artifactStoreService);
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/App.java]: it should be `initializeCronJobs` instead of `initializeBasedOnConfig`

**Review by anandjakhaniya — APPROVED** [review]
Approved after minor fix

File: backend/src/main/java/ai/verta/modeldb/App.java (hunk 1 above)
Comment (conradoverta) [reply backend/src/main/java/ai/verta/modeldb/App.java]: Could you suggest the change via github? It's faster to fix that way =) I can open the IDE if too hard

Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/App.java]:
```suggestion
      CronJobUtils.initializeCronJobs(
          config, services.authService, services.roleService, services.artifactStoreService);
```

Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/App.java]: Done, Sorry for that.

**Pull request merged**